types/src/validator_verifier.rs

```diff
@@ -321,6 +321,7 @@ impl ValidatorVerifier {
     pub fn check_voting_power<'a>(
         &self,
         authors: impl Iterator<Item = &'a AccountAddress>,
+        check_quorum: bool,
```

Comment:
nit: f+1 and 2f+1 are both quorums, no?

may be `check_strong_quorum` or `check_supermajority_quorum`?

Reply:
I don't think f+1 is a "quorum" but I might be wrong

Reply:
@sasha8 what do you think?

Reply:
nvm, just saw the comment in the other PR.